roles/ovirt-ova-pack/files/pack_ova.py

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/python
+
+import os
+import io
+import tarfile
+import sys
+import time
+
+BLOCKSIZE = 512
+NUL = "\0"
+buf = bytearray(4096)
+
+def createTarInfo(name, size):
+    info = tarfile.TarInfo(name)
+    info.size = size
+    info.mtime = time.time()
+    return info
+
+def padToBlockSize(file, size):
+    remainder = size % BLOCKSIZE
+    if remainder > 0:
+        file.write(NUL * (BLOCKSIZE - remainder))
+
+ova_path = sys.argv[1]
+print "opening for write: %s" % ova_path
```

Comment:
Can we support both Python3 and Python2 please?

Reply:
done